doc/rados/operations/health-checks.rst

```diff
@@ -28,162 +28,161 @@ Monitor
-``mon_clock_drift_allowed`` threshold can also be increased, but this
+``mon_clock_drift_allowed`` threshold can also be increased. However, this
 value must stay significantly below the ``mon_lease`` interval in
-order for monitor cluster to function properly.
+order for the monitor cluster to function properly.
 
 MON_MSGR2_NOT_ENABLED
 _____________________
 
-The :confval:`ms_bind_msgr2` option is enabled but one or more monitors is
-not configured to bind to a v2 port in the cluster's monmap.  This
-means that features specific to the msgr2 protocol (e.g., encryption)
-are not available on some or all connections.
+The :confval:`ms_bind_msgr2` option is enabled but one or more monitors are
+not configured to bind to a v2 port in the cluster's monmap. This
+means that features specific to the msgr2 protocol (for example, encryption)
+are unavailable on some or all connections.
 
-In most cases this can be corrected by issuing the command:
+In most cases this can be corrected by running the following command:
 
 .. prompt:: bash $
 
    ceph mon enable-msgr2
 
-That command will change any monitor configured for the old default
-port 6789 to continue to listen for v1 connections on 6789 and also
-listen for v2 connections on the new default 3300 port.
+After this command is run, any monitor configured to listen on the old default
+port (6789) will continue to listen for v1 connections on 6789 and begin to
+listen for v2 connections on the new default port 3300.
 
-If a monitor is configured to listen for v1 connections on a non-standard port (not 6789), then the monmap will need to be modified manually.
+If a monitor is configured to listen for v1 connections on a non-standard port (that is, a port other than 6789), then the monmap will need to be modified manually.
 
 
 MON_DISK_LOW
 ____________
 
-One or more monitors is low on disk space.  This alert triggers if the
-available space on the file system storing the monitor database
-(normally ``/var/lib/ceph/mon``), as a percentage, drops below
+One or more monitors are low on disk space. This health check is raised if the
+percentage of available space on the file system used by the monitor database
+(normally ``/var/lib/ceph/mon``) drops below the percentage value
 ``mon_data_avail_warn`` (default: 30%).
 
-This may indicate that some other process or user on the system is
-filling up the same file system used by the monitor.  It may also
-indicate that the monitors database is large (see ``MON_DISK_BIG``
+This alert might indicate that some other process or user on the system is
+filling up the file system used by the monitor. It might also
+indicate that the monitor database is too large (see ``MON_DISK_BIG``
 below).
 
-If space cannot be freed, the monitor's data directory may need to be
-moved to another storage device or file system (while the monitor
-daemon is not running, of course).
+If space cannot be freed, the monitor's data directory might need to be
```

Comment:
Or the filesystem enlarged in-situ, which is possible in some circumstances.

Reply:
unfinished business